docs/source/examples.rst

```diff
@@ -0,0 +1,128 @@
+========
+Examples
+========
+
+The following examples are based on testing a SLES image using
+the test suite provided from the Open Build Service. For more information
+on installing the test suite see the :doc:`tests` documentation.
+
+Launch & Test a new instance
+============================
+
+To launch and test a new instance of an image provide the `--image-id` or
+`-i` option.
+
+The image ID will look different for all cloud providers. See the examples
```

Comment:
s/providers/frameworks/

We do not want to continue to perpetuate the "provider" wording when we are trying to get rid of it.

Reply:
ack